packages/cli/src/commands/generate/cell/__tests__/cell.test.js

```diff
@@ -438,6 +454,67 @@ test('generates list cells if name is plural', () => {
   expect(listInferredFromName[CELL_PATH]).toMatchSnapshot()
 })
 
+test('TypeScript: generates list cells if list flag passed in', () => {
+  const CELL_PATH = path.normalize(
+    '/path/to/project/web/src/components/BazingaCell/BazingaCell.tsx'
+  )
+
+  const TEST_PATH = path.normalize(
+    '/path/to/project/web/src/components/BazingaCell/BazingaCell.test.tsx'
+  )
+
+  const STORY_PATH = path.normalize(
+    '/path/to/project/web/src/components/BazingaCell/BazingaCell.stories.tsx'
+  )
+
+  const MOCK_PATH = path.normalize(
+    '/path/to/project/web/src/components/BazingaCell/BazingaCell.mock.ts'
+  )
+
+  // Check the file names
+  expect(Object.keys(findDataByIdTypeScript)).toEqual([
+    MOCK_PATH,
+    TEST_PATH,
+    STORY_PATH,
+    CELL_PATH,
+  ])
+
+  // Check the contents
+  expect(findDataByIdTypeScript[CELL_PATH]).toMatchSnapshot()
+  expect(findDataByIdTypeScript[TEST_PATH]).toMatchSnapshot()
+  expect(findDataByIdTypeScript[STORY_PATH]).toMatchSnapshot()
+  expect(findDataByIdTypeScript[MOCK_PATH]).toMatchSnapshot()
+})
+
+test('TypeScript: generates list cells if name is plural', () => {
+  const CELL_PATH = path.normalize(
+    '/path/to/project/web/src/components/MembersCell/MembersCell.tsx'
+  )
+
+  const TEST_PATH = path.normalize(
+    '/path/to/project/web/src/components/MembersCell/MembersCell.test.tsx'
+  )
+
+  const STORY_PATH = path.normalize(
+    '/path/to/project/web/src/components/MembersCell/MembersCell.stories.tsx'
+  )
+
+  const MOCK_PATH = path.normalize(
+    '/path/to/project/web/src/components/MembersCell/MembersCell.mock.ts'
+  )
+
+  // Check the file names
+  expect(Object.keys(listInferredFromNameTypeScript)).toEqual([
+    MOCK_PATH,
+    TEST_PATH,
+    STORY_PATH,
+    CELL_PATH,
+  ])
+
+  // Check the contents
+  expect(listInferredFromNameTypeScript[CELL_PATH]).toMatchSnapshot()
```

Comment:
Checking now; do we want the other tests here too?

```suggestion
  expect(listInferredFromNameTypeScript[CELL_PATH]).toMatchSnapshot()
  expect(listInferredFromNameTypeScript[TEST_PATH]).toMatchSnapshot()
  expect(listInferredFromNameTypeScript[STORY_PATH]).toMatchSnapshot()
  expect(listInferredFromNameTypeScript[MOCK_PATH]).toMatchSnapshot()
```

Reply:
Nvm we're ok; got the full context now.